### PR TITLE
Bindu | BAH-1051 | add support for mysql 5.6/5.7 installation with bahmni

### PR DESCRIPTION
**Pull request by binduak**

### CODE REVIEW

File: roles/bahmni-emr/templates/mysql56.repo.j2 [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[mysql56-community]
+name=MySQL 5.6 Community Server
+baseurl=http://repo.mysql.com/yum/mysql-5.6-community/el/7/x86_64/
+enabled=1
+gpgcheck=0
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-mysql
```
Comment (angshu) [inline roles/bahmni-emr/templates/mysql56.repo.j2:6]: this is just for example sake? or are you actually placing the file there?

File: roles/mysql/templates/mysql56.repo.j2 [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[mysql56-community]
+name=MySQL 5.6 Community Server
+baseurl=http://repo.mysql.com/yum/mysql-5.6-community/el/7/x86_64/
+enabled=1
+gpgcheck=0
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-mysql
```
Comment (angshu) [inline roles/mysql/templates/mysql56.repo.j2:6]: same question as above. Example or actual file?

File: roles/bahmni-emr/templates/mysql56.repo.j2 (hunk 1 above)
Comment (binduak) [reply roles/bahmni-emr/templates/mysql56.repo.j2:6]: @angshu its not an example..earlier we used to have only one mysql.repo.j2 specific to mysql 5.7. But here we have separate files each for mysql 5.6 and mysql 5.7.

File: roles/mysql/templates/mysql56.repo.j2 (hunk 2 above)
Comment (binduak) [reply roles/mysql/templates/mysql56.repo.j2:6]: reply is same as above

**Comment by binduak** [comment]
Merging PR as I got the confirmation from @angshu !

**Pull request merged**